index.html

```diff
@@ -1,44 +1,213 @@
+            <li class="lang3">JavaScript</li>
+          </ul>
+          <button class="btn btnProj">
+            <div class="btnText">See this project</div>
+            <div class="btnIcon">
+              <img src=".png/ArrowR.png" alt="arrow-icon" />
+            </div>
+          </button>
+        </div>
+      </div>
+
+      <div class="imgContainer">
+        <img src=".png/Rectangle6.png" alt="laptop-human-hands" />
+      </div>
+      <div class="infoBox">
+        <div class="projInfo">
+          <h3>Project name goes here</h3>
+          <ul class="langList">
+            <li class="lang1">HTML/CSS</li>
+            <li class="lang2">Ruby on Rails</li>
+            <li class="lang3">JavaScript</li>
+          </ul>
+          <button class="btn btnProj">
+            <div class="btnText">See this project</div>
+            <div class="btnIcon">
+              <img src=".png/ArrowR.png" alt="arrow-icon" />
+            </div>
+          </button>
+        </div>
+      </div>
+    </section>
+    <section class="aboutMyself">
+      <div class="aboutSection">
+        <h1 class="aboutHeader">About</h1>
+        <p class="pabout">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ea eiusmod
+        </p>
+        <button class="btn myCv">
+          <div class="btnTextGet">Get my CV</div>
+          <div class="btnIconDownload">
+            <img src=".png/ic_download.png" alt="download-icon" />
+          </div>
+        </button>
+        <div class="aboutCards">
+          <h4 class="langCardHeader">Languages</h4>
+          <ul class="langCardList">
+            <li class="laCardListItem">JavaScript (ES5/ES6)</li>
+            <li class="laCardListItem">Ruby</li>
+            <li class="laCardListItem">HTML</li>
+            <li class="laCardListItem">CSS</li>
+          </ul>
+        </div>
+        <div class="framworksCard">
+          <h4 class="framworksCardHeader">Frameworks</h4>
+          <ul class="framworksCardList">
+            <li class="laCardListItem">Bootstrap</li>
+            <li class="laCardListItem">Ruby on rails</li>
+            <li class="laCardListItem">RSpec</li>
+            <li class="laCardListItem">Capibara</li>
+            <li class="laCardListItem">Selenium</li>
+          </ul>
+        </div>
+        <div class="skillsCard">
+          <h4 class="skillCardHeader">Skills</h4>
+          <ul class="skillCardList">
+            <li class="laCardListItem">Codekit</li>
+            <li class="laCardListItem">Codepen</li>
+            <li class="laCardListItem">GitHub</li>
+            <li class="laCardListItem">GitLAb</li>
+            <li class="laCardListItem">Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section class="contactMe"></section>
```

Comment:
- [ ] Kindly consider wrapping all the cards in a section and use `display:grid` as in the project requirements.